fastlane/spec/actions_specs/download_dsyms_spec.rb

```diff
@@ -74,6 +75,36 @@
         end
       end
 
+      context 'when version is live' do
+        before do
+          # live
+          allow(app).to receive(:live_version).and_return(live_version)
+          allow(live_version).to receive(:version).and_return('1.0.0')
+          allow(live_version).to receive(:build_version).and_return('42')
+          allow(live_version).to receive(:candidate_builds).and_return([build])
+          allow(build).to receive(:build_version).and_return('42')
+          allow(build).to receive(:upload_date).and_return(1_547_196_482_000)
+          # latest
```

Comment:
I think the `allows` from here and below can be deleted since we really only care about `app.live_version`, right?

Reply:
I'm new to this test setup, so I wasn't sure if I should leave it or not. Because my PR was to fix getting the live version, if there is already an edit_version *and* even a newer build train, which was the initial issue we discovered. should I remove it or keep it?

Reply:
I added a commit to remove this for you! It can be removed because the latest code is being executed in a different if statement during the test so nothing needs to get mocked up for that 😊 After the tests run I will get this merged in. Thanks for contribution ❤️

Reply:
cool thx! 🎉